### PR TITLE
refine wren-ai-service logging and setups

### PR DESCRIPTION
**Pull request by cyyeh**
- added some missing logging info based on .env.ai, so in the future we can understand whether users fill in wrong information
- fixed issues for the command `make dev-up` due to new version of `wren-engine`
- add `azure_deployment` to `azure_openai.py` for embedder and provider implementation

### CODE REVIEW

File: wren-ai-service/Makefile [hunk 1]
```diff
@@ -12,6 +12,7 @@ dev-down:
 
 ## wren-ai-service related ##
 start:
+	cp .env.example .env
```
Comment (paopa) [inline wren-ai-service/Makefile:15]: I do believe we could remove the command and instead modify the function for loading the environment variables. What do you think?

**Review by paopa — COMMENTED** [review]
Have some questions. others good to me.

File: wren-ai-service/src/providers/llm/azure_openai.py [hunk 2]
```diff
@@ -130,10 +132,11 @@ def __init__(
...
         self,
-        model_kwargs: Dict[str, Any] = orjson.loads(
-            os.getenv("GENERATION_MODEL_KWARGS", "{}")
-        )
-        or GENERATION_MODEL_KWARGS,
+        model_kwargs: Dict[str, Any] = (
+            orjson.loads(os.getenv("GENERATION_MODEL_KWARGS"))
+            if os.getenv("GENERATION_MODEL_KWARGS")
+            else GENERATION_MODEL_KWARGS
+        ),
```
[the remark this answers is not in the document]
Comment (cyyeh) [reply wren-ai-service/src/providers/llm/azure_openai.py:140]: synced

File: wren-ai-service/Makefile [hunk 3]
```diff
@@ -42,9 +43,6 @@ prepare-wren-engine:
 	mkdir -p tools/dev/etc/mdl
 	echo "{\"catalog\": \"test_catalog\", \"schema\": \"test_schema\", \"models\": []}" \\
 		> tools/dev/etc/mdl/sample.json
-
-psql:
-	docker exec -it wren-engine-1 bash launch-cli.sh
```
[the remark this answers is not in the document]
Comment (cyyeh) [reply wren-ai-service/Makefile:47]: synced

File: wren-ai-service/Makefile (hunk 1 above)
Comment (cyyeh) [reply wren-ai-service/Makefile]: synced

**Review by paopa — APPROVED** [review]
LGTM

**Pull request merged**